### PR TITLE
Trials api JSON

### PR DESCRIPTION
**Pull request by susanX**
-I added Eligibility and Exclusion info.
see record 1 line 68. Also added to all 35 records.
-Within the current structure "IDInfo" object under NCTID I have added the actual title. See line 14. Also added for 35 records.
-I removed Record 6 as it had 1000s of line of addresses, 99% not UK.
- Comment: the none UK address could be remove from the file.
- See Figma Mobile for the reduced icons and new Eligibility info.
- The file validates as json.

### CODE REVIEW

**Comment by susanX** [comment]
https://api.myjson.com/bins/118mwa  now out of date.
I have made a small temporary api for testing the json with just 13 records on it.
It can be fetched from the url above.

**Comment by susanX** [comment]
Changes to Eligibility inclusion and exclusion approved.

**Comment by susanX** [comment]
Approved.

**Review by susanX — COMMENTED** [review]
ok

File: Results-10-10-19.json [hunk 1]
```diff
@@ -0,0 +1,12580 @@
+{
```
Comment (MohammedYehia) [inline Results-10-10-19.json:1]: I think we don't need this file anymore as there is the json file on the dummydata folder, right?

**Review by MohammedYehia — CHANGES_REQUESTED** [review]

File: Results-10-10-19.json (hunk 1 above)
Comment (susanX) [reply Results-10-10-19.json:1]: We may still need to set up a fetch request depending on Jon

**Review by MohammedYehia — APPROVED** [review]

**Pull request merged**